package.json

```diff
@@ -97,6 +98,7 @@
   },
   "peerDependencies": {
     "preact": "7.1.0",
-    "preact-compat": "3.13.1"
+    "preact-compat": "3.13.1",
```

Comment:
it is a peerDependency which means it is not installed automatically but you can install it by yourself. In general there should also be react in this list and you could choose which peer u want to fulfill.

So no you don't.